src/parser/scanner.lex

```diff
@@ -342,8 +346,12 @@ FORMAT                      ([Ff][Oo][Rr][Mm][Aa][Tt])
 {SUBGRAPH}                  { return TokenType::KW_SUBGRAPH; }
 {CONTAINS}                  { return TokenType::KW_CONTAINS; }
 {NOT_CONTAINS}              { return TokenType::KW_NOT_CONTAINS; }
+{STARTS}                    { return TokenType::KW_STARTS;}
```

Comment:
There has `KW_STARTS` AND `KW_WITH`, why exist `NOT_STARTS_WITH ` too.

Reply:
If we use 2 separate keywords to perform one operator there will be a reduce/shift error occurring when loading the parser.yy, which causes a compilation failure.